pkg/cmd/gitprovider/add.go

```diff
@@ -38,6 +39,14 @@ var GitProviderAddCmd = &cobra.Command{
 			return
 		}
 
+		// TODO: Remove this when we starts to encode git clone url
+		if *gitProviderData.Id == "aws-codecommit" {
```

Comment:
This can be removed when https://github.com/daytonaio/daytona/pull/841 is merged.

Leaving this comment here until it's resolved.